### DISCUSSION TITLE
i'm getting this error while creating eks cluster

### DISCUSSION THREAD

**Discussion by muralikatikamarati** [learndevopsonline/learndevopsonline#1913] [Q&A]
![eks error today](https://github.com/learndevopsonline/learndevopsonline/assets/121479020/426a64d7-5fa5-4336-b96b-7820795fbfc0)
 i'm getting this error while creating eks cluster 

Error: waiting for EKS Node Group (my-eks-cluster:my-node-group) create: unexpected state 'CREATE_FAILED', wanted target 'ACTIVE'. last error: i-00ed7242c392e6714, i-05dafd1875bc6156d: NodeCreationFailure: Instances failed to join the kubernetes cluster
│
│   with aws_eks_node_group.node_group,
│   on main.tf line 60, in resource "aws_eks_node_group" "node_group":
│   60: resource "aws_eks_node_group" "node_group" {

how can i fix this issue

i tried but not able to fix this issue

github link
https://github.com/muralikatikamarati/VPC-NETWORK-2

**Comment by chaithanya1812** [comment]
Hi @muralikatikamarati 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/5982dfc5-8786-492e-bb76-00304492becb)
